assignments/ex2/rubyScript.rb

```diff
@@ -0,0 +1,35 @@
+
+require 'rubygems'
+require 'json'
+
```

Comment:
You need to use the  environment variables and some external dependencies in your code